pure_pursuit_wrapper/launch/TrajectoryPlan.message

```diff
@@ -0,0 +1,3 @@
+'{ header: auto, pose:
+  position: x: 0.0, y: 0.0, z: 0.0
+  orientation: x: 0.0, y: 0.0, w: 0.0 }'
```

Comment:
you should have a z component with this orientation

Reply:
File pushed by accident, removed